cmd/krel/cmd/changelog_data_test.go

```diff
@@ -412,7 +412,7 @@ Prior to 1.17 and for existing nodes created by <code>kubeadm init</code> where
 <li>Add <code>scheduler_goroutines</code> metric to track number of kube-scheduler binding and prioritizing goroutines (<a href="https://github.com/kubernetes/kubernetes/pull/83535">#83535</a>, <a href="https://github.com/wgliang">@wgliang</a>)</li>
 <li>Adding initial EndpointSlice metrics. (<a href="https://github.com/kubernetes/kubernetes/pull/83257">#83257</a>, <a href="https://github.com/robscott">@robscott</a>)</li>
 <li>Adds a metric <code>apiserver_request_error_total</code> to kube-apiserver. This metric tallies the number of <code>request_errors</code> encountered by verb, group, version, resource, subresource, scope, component, and code. (<a href="https://github.com/kubernetes/kubernetes/pull/83427">#83427</a>, <a href="https://github.com/logicalhan">@logicalhan</a>)</li>
-<li>A new <code>kubelet_preemptions</code> metric is reported from Kubelets to track the number of preemptions occuring over time, and which resource is triggering those preemptions. (<a href="https://github.com/kubernetes/kubernetes/pull/84120">#84120</a>, <a href="https://github.com/smarterclayton">@smarterclayton</a>)</li>
+<li>A new <code>kubelet_preemptions</code> metric is reported from Kubelets to track the number of preemptions occurring over time, and which resource is triggering those preemptions. (<a href="https://github.com/kubernetes/kubernetes/pull/84120">#84120</a>, <a href="https://github.com/smarterclayton">@smarterclayton</a>)</li>
```

Comment:
I think it's not possible to fix that because we're retrieving the changelog from [here](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.17/release-notes-draft.md#added-metrics). That’s why I added the `// nolint: misspell` in line 330. :)

Reply:
ahh did not see that :/

so should I close the entire pr?

Reply:
No I recommend to just revert that line, then the tests should pass :)